.vsts-pr.yaml

```diff
@@ -104,6 +104,9 @@ jobs:
       fsharpqa_release:
         _configuration: Release
         _testKind: testFSharpQA
```

Comment:
A few lines up change `maxParallel` to `5`; this will enable all Windows legs to run at the same time.